res/http.php

```diff
@@ -27,8 +30,11 @@
  * HTTP stack.
  */
 return pipe([
+    BlackfireMiddleware::class,
+    NewRelicMiddleware::class,
```

Comment:
Since Blackfire and New Relic consider that 1 HTTP request == 1 PHP process, they will not work correctly out of the box.

Those 2 middlewares activate manually Blackfire and New Relic for each HTTP request.

Those 2 middlewares could be moved to their own open source projects though.